StringUtils.cpp

```diff
@@ -25,14 +25,12 @@ std::string remQuotes(const std::string& in)
 
 std::string addQuotes(const std::string& in)
 {
-	const auto length = in.size();
-
-	if (length == 0)
+	if (in.empty())
 	{
 		return "\"\"";
 	}
 
-	if (in[0] == '"' && in[length - 1] == '"')
+	if (in.starts_with('"') && in.ends_with('"'))
```

Comment:
logic is broken for the case in is a single char with a quote. Should have size>2 for that edge case.

Reply:
But first, add a test demonstrating the issue.

Reply:
What should be returned when the string is `"`? `""`?

Reply:
Can't happen, same as the single quote example since we start inQuotes and don't stop loading until quote is closed. However, that's parser, and if we ever change that behavior, this will silently fail.
If lexeme doesn't end with a quote but starts with one, leave as is.

Reply:
Please check 85213a4

Reply:
The tests look good to me